wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
+|-----------------------------|----------------------------------------------------------------|
+| Stable (Latest)             | Public build.                                                  |
+| Stable (Fallback)           | Older public build in case any major bugs come up.             |
+| Beta                        | Get new features early, but guaranteed to have bugs.           |
+| Cutting Edge (Experimental) | Get new features the instant they're complete, lots more bugs. |
+
+#### Other
+
+| Name               | Effect                                                    |
+|--------------------|-----------------------------------------------------------|
+| osu! is up-to-date | Clicking this will force osu! to check for updates again. |
+| Open osu! folder   | Open osu! folder...                                       |
+
+## Graphics section
 
 ### Renderer
 
-<table style="width:36%;">
-<colgroup>
-<col width="12%" />
-<col width="24%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Frame Limiter: (controls the maximum number of frames displayed per second)</p></td>
-<td><p>VSync</p>
-<hr>
-<p>120fps [Default]</p>
-<hr>
-<p>240fps</p>
-<hr>
-<p>Unlimited (gameplay)</p>
-<hr>
-<p><strong>Note:</strong> For laptop users, it is recommended to limit your frames to prevent overheating <a href="http://osu.ppy.sh/forum/t/82605">as mentioned in this thread</a>. Also, mind your electricity usage bill.</p></td>
-</tr>
-<tr class="even">
-<td><p>Show FPS Counter</p></td>
-<td><p>Show subtle Frame-Per-Second counter at bottom-right of the screen</p></td>
-</tr>
-<tr class="odd">
-<td><p>Compatibility Mode</p></td>
-<td><p>Reverts the way how osu! renders<sup>[Citation Needed]</sup>; only if you run into troubles running osu! (Forces osu! to restart)</p>
-<ul>
-<li>Compatibility mode may increase overall input latency and reduce gameplay performance.</li>
-</ul></td>
-</tr>
-<tr class="even">
-<td><p>Reduce dropped frames</p></td>
-<td><p>Users who experienced frequent dropped/stutter frame rate should try turning this on. May reduce framerate slightly.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Detect performance issues</p></td>
-<td><p>osu! will actively search and notify you for programs/clients that may affect your game performance.</p>
-<ul>
-<li>Known issues: <a href="http://justgetflux.com/">f.lux</a>.</li>
-</ul></td>
-</tr>
-</tbody>
-</table>
+| Name                      | Effect                                                                           |
+|---------------------------|----------------------------------------------------------------------------------|
+| Frame Limiter             | VSync<br>120fps (Default)<br>240fps<br>Unlimited (gameplay)                      |
+| Show FPS Counter          | Show FPS counter at the bottom-right corner of the screen.                       |
```

Comment:
change `Show FPS counter at the bottom-right corner of the screen.` to
`Display the FPS counter at the bottom-right corner of the screen.`